amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2092,6 +2097,417 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  ## Hold containers
+  '/hold-containers/{holdContainerId}/holds':
```

Comment:
I don't think we want to add the concept of hold containers. All holds can go in the same place. We could run this by Lilian if you like though?

Reply:
Ok - I think this is resolved now we have the `file-plan-id` in the path.